examples/modbus/modbus_main.c

```diff
@@ -432,20 +432,22 @@ int main(int argc, FAR char *argv[])
 
   /* Handle command line arguments */
 
-  g_modbus.quit = false;
+  g_modbus.quit = true;
```

Comment:
why `quit` is defaulted to `true`?

Reply:
Because now I use it to decide if the program needs to finishes normally or keep waiting. The only option where it keeps waiting is when user passes "modbus -e". Initially I was "g_modbus.quit = true" inside each switch/case option except the case 'e'. So I realized it was better to invert the value to simplify the code.

Reply:
Ah ok, I forgot to update the comment

Reply:
wait, it was a mistake! Probably I inverted in the top, removed the others quit = true and inverted that from false to true...